pkg/services/folder/folderimpl/folder_test.go

```diff
@@ -378,6 +387,8 @@ func TestIntegrationNestedFolderService(t *testing.T) {
 			require.NoError(t, err)
 			_ = insertTestDashboard(t, serviceWithFlagOn.dashboardStore, "dashboard in parent", orgID, parent.ID, "prod")
 			_ = insertTestDashboard(t, serviceWithFlagOn.dashboardStore, "dashboard in subfolder", orgID, subfolder.ID, "prod")
+			_ = createRule(t, alertStore, parent.UID, "parent alert")
+			_ = createRule(t, alertStore, subfolder.UID, "sub alert")
```

Comment:
I had considered it, but I wanted to avoid having to create a more complex alert rule. There's [this alert rule validation step](https://github.com/grafana/grafana/blob/main/pkg/services/ngalert/store/alert_rule.go#L535) I was trying to bypass.

Also there's precedent for this. ngalert store also has a similar function for [creating alerts for testing purposes](https://github.com/grafana/grafana/blob/main/pkg/services/ngalert/store/alert_rule_test.go#L223). Happy to use the function you mentioned if you think it would be worth creating a full alert rule.

Reply:
it's just a test, so I'm fine with leaving it as is